.github/workflows/ci.yml

```diff
@@ -56,6 +56,18 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
+      - name: geninstaller Cross Check
```

Comment:
Use a cross-check for now. May want to remove it when `cmd/geninstaller` is completely deprecated.